src/main/scala/chisel3/util/BlackBoxUtils.scala

```diff
@@ -29,3 +32,15 @@ trait HasBlackBoxInline extends BlackBox {
     chisel3.experimental.annotate(anno)
   }
 }
+
+trait HasBlackBoxPath extends BlackBox {
+  self: BlackBox =>
+
+  def addPath(blackBoxPath: String): Unit = {
```

Comment:
ScalaDoc here too. Probably worth mentioning that the working directory is
not likely to be the same as the target directory